projects/plugins/jetpack/modules/masterbar/masterbar/class-masterbar.php

```diff
@@ -184,6 +196,12 @@ class_exists( 'Jetpack_AMP_Support' )
 			// Override Notification module to include RTL styles.
 			add_action( 'a8c_wpcom_masterbar_enqueue_rtl_notification_styles', '__return_true' );
 		}
+
+		// Hides and replaces the language dropdown for the current user.
+		if ( empty( $_GET['user_id'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
I've disabled this nonce warning because we're not actually processing form data. But let me know if I've missed something.

`201 | WARNING | Processing form data without nonce verification.`

Also disabled here:

https://github.com/Automattic/jetpack/blob/5648ec24f73fd69b82603d7f9cc7a38484f77bab/projects/plugins/jetpack/modules/masterbar/masterbar/class-masterbar.php#L204